pkg/tunnelserver/tunnel.go

```diff
@@ -279,7 +282,13 @@ func (t *Authorizer) authorizeCluster(cluster *v3.Cluster, inCluster *cluster, r
 	}
 
 	changed := false
-	if cluster.Status.Driver == "" {
+
+	driver, err := provisioner.GetDriver(cluster, t.KontainerDriverLister)
```

Comment:
The old code would only set the cluster.Status.Driver field if is empty.  This will now always "reset" it, is that intended? Additionally will always set changed to true which may result in a etcd write per node checkin.

Reply:
@ibuildthecloud 

> This will now always "reset" it, is that intended?

It will only reset it if the driver is empty - line 290 has this check. The only diff this pr brings is  - `provisioner.GetDriver` call is more deterministic, and will never return empty value if RKE (or any other k8s provider config) is returned.

Reply:
Thanks